test/jdk/java/lang/StackWalker/Basic.java

```diff
@@ -111,6 +111,11 @@ public static void testMethodSignature() throws Exception {
         this.depth = depth;
     }
 
+    /** For TestNG */
+    public Basic() {
+        depth = 0;
```

Comment:
`depth` is `final`, so it needs to be assigned. but this could be replaced with `this(0)`

Reply:
Ah, missed the final on depth :-)

Reply:
Any particular strengths associated with `this(0)` as opposed to the actual assignment? Aside from the brevity it provides of course

Reply:
My personal preference is to chain constructors whenever possible, ideally ending up in a single constructor that all other constructors call. This way the initialization logic is in a single place - instead of being duplicate all over.